<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lcQPreferencesDialog</class>
 <widget class="QDialog" name="lcQPreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Default projects folder:</string>
         </property>
         <property name="buddy">
          <cstring>projectsFolder</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Custom parts library:</string>
         </property>
         <property name="buddy">
          <cstring>partsLibrary</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Mouse sensitivity:</string>
         </property>
         <property name="buddy">
          <cstring>mouseSensitivity</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSlider" name="mouseSensitivity">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="pageStep">
          <number>5</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QCheckBox" name="checkForUpdates">
         <property name="text">
          <string>Check for updates on startup</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="2">
        <widget class="QCheckBox" name="centimeterUnits">
         <property name="text">
          <string>Centimeter units</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0" colspan="2">
        <widget class="QCheckBox" name="noRelativeSnap">
         <property name="text">
          <string>Don't allow relative snap</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0" colspan="2">
        <widget class="QCheckBox" name="fixedDirectionKeys">
         <property name="text">
          <string>Fixed direction keys</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="projectsFolder"/>
         </item>
         <item>
          <widget class="QToolButton" name="projectsFolderBrowse">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="partsLibrary"/>
         </item>
         <item>
          <widget class="QToolButton" name="partsLibraryBrowse">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Default author name:</string>
         </property>
         <property name="buddy">
          <cstring>authorName</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="authorName"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>POV-Ray Executable</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>LGEO Path</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLineEdit" name="povrayExecutable"/>
         </item>
         <item>
          <widget class="QToolButton" name="povrayExecutableBrowse">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLineEdit" name="lgeoPath"/>
         </item>
         <item>
          <widget class="QToolButton" name="lgeoPathBrowse">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabRendering">
      <attribute name="title">
       <string>Rendering</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QCheckBox" name="baseGrid">
         <property name="text">
          <string>Base grid</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="antiAliasing">
         <property name="text">
          <string>Anti-aliasing</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="edgeLines">
         <property name="text">
          <string>Edge lines</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="fastRendering">
         <property name="text">
          <string>Fast rendering</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="enableLighting">
         <property name="text">
          <string>Enable lighting</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineWidth">
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="antiAliasingSamples">
         <item>
          <property name="text">
           <string>2x</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4x</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8x</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="axisIcon">
         <property name="text">
          <string>Axis icon</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="gridUnits">
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>units</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>width</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCategories">
      <attribute name="title">
       <string>Categories</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Parts Library Categories</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QSplitter" name="splitter">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QTreeWidget" name="categoriesTree">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="rootIsDecorated">
              <bool>false</bool>
             </property>
             <property name="uniformRowHeights">
              <bool>true</bool>
             </property>
             <property name="itemsExpandable">
              <bool>false</bool>
             </property>
             <property name="expandsOnDoubleClick">
              <bool>false</bool>
             </property>
             <column>
              <property name="text">
               <string>Categories</string>
              </property>
             </column>
            </widget>
            <widget class="QTreeWidget" name="partsTree">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>2</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
             <property name="rootIsDecorated">
              <bool>false</bool>
             </property>
             <property name="uniformRowHeights">
              <bool>true</bool>
             </property>
             <property name="itemsExpandable">
              <bool>false</bool>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <property name="columnCount">
              <number>2</number>
             </property>
             <attribute name="headerShowSortIndicator" stdset="0">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Number</string>
              </property>
             </column>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="importCategories">
              <property name="text">
               <string>Import...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="exportCategories">
              <property name="text">
               <string>Export...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="resetCategories">
              <property name="text">
               <string>Reset...</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Category</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="newCategory">
            <property name="text">
             <string>New...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editCategory">
            <property name="text">
             <string>Edit...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteCategory">
            <property name="text">
             <string>Delete...</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabKeyboard">
      <attribute name="title">
       <string>Keyboard</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Keyboard Shortcuts</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QTreeWidget" name="commandList">
                <property name="rootIsDecorated">
                 <bool>false</bool>
                </property>
                <property name="uniformRowHeights">
                 <bool>true</bool>
                </property>
                <column>
                 <property name="text">
                  <string>Command</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Shortcut</string>
                 </property>
                </column>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QPushButton" name="shortcutsImport">
                  <property name="text">
                   <string>Import...</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="shortcutsExport">
                  <property name="text">
                   <string>Export...</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="shortcutsReset">
                  <property name="text">
                   <string>Reset...</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6"/>
           </item>
           <item>
            <widget class="QGroupBox" name="shortcutGroup">
             <property name="title">
              <string>Shortcut</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Key sequence:</string>
                </property>
                <property name="buddy">
                 <cstring>shortcutEdit</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="shortcutEdit"/>
              </item>
              <item>
               <widget class="QPushButton" name="shortcutAssign">
                <property name="text">
                 <string>Assign</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="shortcutRemove">
                <property name="text">
                 <string>Remove</string>
                </property>
               </widget>
              </item>
             </layout>
             <zorder>shortcutEdit</zorder>
             <zorder>label_7</zorder>
             <zorder>shortcutAssign</zorder>
             <zorder>shortcutRemove</zorder>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>authorName</tabstop>
  <tabstop>projectsFolder</tabstop>
  <tabstop>projectsFolderBrowse</tabstop>
  <tabstop>partsLibrary</tabstop>
  <tabstop>partsLibraryBrowse</tabstop>
  <tabstop>povrayExecutable</tabstop>
  <tabstop>povrayExecutableBrowse</tabstop>
  <tabstop>lgeoPath</tabstop>
  <tabstop>lgeoPathBrowse</tabstop>
  <tabstop>mouseSensitivity</tabstop>
  <tabstop>checkForUpdates</tabstop>
  <tabstop>centimeterUnits</tabstop>
  <tabstop>noRelativeSnap</tabstop>
  <tabstop>fixedDirectionKeys</tabstop>
  <tabstop>antiAliasing</tabstop>
  <tabstop>antiAliasingSamples</tabstop>
  <tabstop>edgeLines</tabstop>
  <tabstop>lineWidth</tabstop>
  <tabstop>baseGrid</tabstop>
  <tabstop>gridUnits</tabstop>
  <tabstop>axisIcon</tabstop>
  <tabstop>enableLighting</tabstop>
  <tabstop>fastRendering</tabstop>
  <tabstop>categoriesTree</tabstop>
  <tabstop>partsTree</tabstop>
  <tabstop>importCategories</tabstop>
  <tabstop>exportCategories</tabstop>
  <tabstop>resetCategories</tabstop>
  <tabstop>newCategory</tabstop>
  <tabstop>editCategory</tabstop>
  <tabstop>deleteCategory</tabstop>
  <tabstop>commandList</tabstop>
  <tabstop>shortcutsImport</tabstop>
  <tabstop>shortcutsExport</tabstop>
  <tabstop>shortcutsReset</tabstop>
  <tabstop>shortcutEdit</tabstop>
  <tabstop>shortcutAssign</tabstop>
  <tabstop>shortcutRemove</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>lcQPreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>lcQPreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
